timemachine/fe/single_topology_v3.py

```diff
@@ -177,10 +177,59 @@ def setup_dummy_interactions(
     )
 
 
+def canonicalize_improper_idxs(idxs):
```

Comment:
Nit: annotation
```suggestion
def canonicalize_improper_idxs(idxs) -> Tuple[int, int, int, int]:
```
(since other canonicalize function returns length-4 sequence of type `type(idxs)` and this discards `type(idxs)` and returns 4-tuple)